migration/sql-files/110-trackerquery-to-use-uuid.sql

```diff
@@ -0,0 +1,6 @@
+-- Add a new UUID field to the trackerquery table and let other tables use that instead of the current id
+ALTER TABLE tracker_queries ADD COLUMN trackerquery_id uuid DEFAULT uuid_generate_v4() NOT NULL;
+
+-- Rename primary key of trackers table
+ALTER TABLE tracker_queries DROP COLUMN id CASCADE;
+ALTER TABLE tracker_queries RENAME COLUMN trackerquery_id TO id;
```

Comment:
@DhritiShikhar the `id` should become the primary key for the `tracker_queries` as well, no?

Reply:
added here 2d0d18a